medicines/doc-index-updater/examples/manifests/README.md

```diff
@@ -28,20 +28,16 @@ curl -vvv http://doc-index-updater.localhost/non-existent-route # should be 404
 
 ### Encrypt the redis key as a Sealed Secret
 
-- retrieve the keys for the redis cache from the Azure Portal or the cli:
-
-```bash
-az redis list-keys --resource-group MHRA-dev --name doc-index-updater-dev --output tsv
-```
-
-- create the sealed secret with one of the above keys (replace `<insert key here>` with the key):
+- retrieve the keys for the redis cache from the Azure Portal or the cli, and create the sealed secret with one of the above keys:
```

Comment:
doesn't make sense now, should say something like:

> create the sealed secret with keys for the redis cache retrieved from `az redis` cli,

Reply:
I'm going to update this `manifests` example to point to the relevant manifests in the `deployments` repo. We really only want to maintain these in one place.